### PR TITLE
📜 Add documentation for create eth-app

### PR DESCRIPTION
**Pull request by mj426382**
https://app.asana.com/0/1201906120944847/1201994471245191

https://github.com/paulrberg/create-eth-app

### CODE REVIEW

File: packages/docs/docs/alternative-getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Create Eth App is a great fit for:
+
+* Learning how to write Ethereum-powered apps in a comfortable and feature-rich development environment.
+* Starting new Ethereum-powered single-page React applications without wasting time on copy-pasting boilerplates
+* Creating examples with React for your Ethereum-related libraries and components.
+
+## Code
+
+Source code on Github [here](https://github.com/paulrberg/create-eth-app).
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:131]:
```suggestion
Source code is available on [Github](https://github.com/paulrberg/create-eth-app).
```

File: packages/docs/docs/alternative-getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+## Pros
+
+Create Eth App is a great fit for:
+
+* Learning how to write Ethereum-powered apps in a comfortable and feature-rich development environment.
+* Starting new Ethereum-powered single-page React applications without wasting time on copy-pasting boilerplates
+* Creating examples with React for your Ethereum-related libraries and components.
+
+## Code
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:129]:
```suggestion
## Source code
```

File: packages/docs/docs/alternative-getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+## Start
+
+Just run
+
+```bash yarn
+yarn react-app:start
+```
+
+After that the app works, so open [http://localhost:3000/](http://localhost:3000/).
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:76]:
```suggestion
After that the app starts locally, open [http://localhost:3000/](http://localhost:3000/) to view it in the browser.
```

File: packages/docs/docs/alternative-getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        ├── src
+        │   └── mappings
+        │       ├── tokens.ts
+        │       └── transfers.ts
+        └── subgraph.yaml
+```
+
+## Start
+
+Just run
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:70]:
```suggestion
Just run:
```

File: packages/docs/docs/alternative-getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+
+```bash yarn
+yarn react-app:start
+```
+
+After that the app works, so open [http://localhost:3000/](http://localhost:3000/).
+
+## Frameworks
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:78]: UseDapp doesn't work with any other framework than the default React, so I propose we remove this section from our doc.

File: packages/docs/docs/alternative-getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 9
+id: alternative-getting-started
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:3]:
```suggestion
id: create-eth-app
```

Is the `id ` used or needed?

File: packages/docs/docs/alternative-getting-started.mdx [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+---
+
+#  Getting started with create eth-app
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:6]:
```suggestion
#  Getting started with Create Eth App
```

File: packages/docs/docs/alternative-getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,131 @@
+---
+sidebar_position: 9
```
Comment (rzadp) [inline packages/docs/docs/alternative-getting-started.mdx:2]: Should be below the first `getting started`

**Review by rzadp — CHANGES_REQUESTED** [review]

File: packages/docs/docs/alternative-getting-started.mdx (hunk 5 above)
Comment (mj426382) [reply packages/docs/docs/alternative-getting-started.mdx:78]: Fine

File: packages/docs/docs/alternative-getting-started.mdx (hunk 6 above)
Comment (mj426382) [reply packages/docs/docs/alternative-getting-started.mdx:3]: It is just my style vision:)

**Review by rzadp — APPROVED** [review]

**Pull request merged**